common/utils/src/main/scala/org/apache/spark/SparkThrowableHelper.scala

```diff
@@ -99,6 +107,19 @@ private[spark] object SparkThrowableHelper {
           g.writeStringField("errorClass", errorClass)
           if (format == STANDARD) {
             g.writeStringField("messageTemplate", errorReader.getMessageTemplate(errorClass))
+            errorReader.getBreakingChangeInfo(errorClass).foreach{ breakingChangeInfo =>
+              g.writeObjectFieldStart("breakingChangeInfo")
+              g.writeStringField("migrationMessage",
+                  breakingChangeInfo.migrationMessage.mkString("\n"))
+              breakingChangeInfo.mitigationSparkConfig.foreach{ mitigationSparkConfig =>
+                g.writeObjectFieldStart("mitigationSparkConfig")
```

Comment:
just for my education: this can write JSON array? The JSON writer can recognize duplicated object field names automatically?